utils/utils.js

```diff
@@ -117,6 +117,13 @@ function getJSONTrimmed(context, url, writeKey, callback) {
   xhr.send();
 }
 
+function handleErrorWithBugsnag(error, breadcrumb) {
```

Comment:
We can keep the function name generic to improve extensibility.
```suggestion
function notifyError(error, breadcrumb) {
```